bin/build_project.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+export WORKDIR=`pwd`
+echo "(RE)Building Project ..."
+
+echo "Activating virtual enviroment..."
+source ../env/bin/activate
+
+echo "Installing Python Dependencies..."
+pip install -r requirements.txt
+echo
+CODE=$?
+[ $CODE -ne 0 ] && exit $CODE
+
+echo "Updating NPM packages..."
+npm install
+echo
+CODE=$?
+[ $CODE -ne 0 ] && exit $CODE
+
+echo "Building JS ..."
+ln -s /usr/bin/nodejs /usr/bin/node
```

Comment:
Not sure about this line with link. if it is required on all systems, but on Ubuntu 14.04 I had to do this. So , for now it is OK.